### ISSUE TITLE
min: 0 does not work for type number

### ISSUE THREAD

**Issue by ronny332** [majorleaguesoccer/respectify#10]
Hi,

is this already a known bug?

```
q: {
          dataTypes: ['number'],
          max: 99,
          min: 0
        }
```

does not work for me with the latest version and a minimum of 0. the type/max checking does just not work.

Changing min to 1 or 10 works brilliant with the known "code": "InvalidArgument" message.

I'm currently on my way home, so I'm very sorry for this short bug report. Hopefully I will find the time to extend it this evening.

**Comment by sorensen** [comment]
Thats not good, I will investigate and get a fix out. Thanks for finding this.

**Comment by sorensen** [comment]
@ronny332 thanks for catching this, I have published the fix in `1.3.3`
